diff -ruN st-0.8.1/config.def.h st-0.8.1-mj/config.def.h
--- st-0.8.1/config.def.h	2018-03-20 13:29:59.000000000 -0700
+++ st-0.8.1-mj/config.def.h	2018-07-27 17:33:01.064512367 -0700
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "Liberation Mono:pixelsize=16:antialias=true:autohint=true";
 static int borderpx = 2;
 
 /*
